integreat_cms/api/v3/locations.py

```diff
@@ -71,9 +71,24 @@ def transform_poi_translation(poi_translation: POITranslation) -> dict[str, Any]
 
     poi = poi_translation.poi
 
-    primary_contact = Contact.objects.filter(
-        location=poi, area_of_responsibility=""
-    ).first()
+    contacts = Contact.objects.filter(location=poi).all()
+    contact_data = []
+    for contact in contacts:
+        contact_data.append(
+            {
+                "id": contact.id,
+                "area_of_responsibility": contact.area_of_responsibility
+                if contact.area_of_responsibility
+                else None,
+                "name": contact.name,
+                "email": contact.email,
+                "phone_number": contact.phone_number,
+                "website": contact.website,
+            }
+        )
+
+    # for transitional period
```

Comment:
Please link an issue here which should remove the comment (see the suggested Todo comment format of ruff, for example: https://docs.astral.sh/ruff/rules/missing-todo-link/).